common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -193,6 +193,10 @@ def modulestore():
             settings.MODULESTORE['default'].get('OPTIONS', {})
         )
 
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX'):
+            from ccx.modulestore import CCXModulestoreWrapper
```

Comment:
I think circular import is because `ccx/utils.py` is importing `modulestore.django`.